client/containers/DashboardOverview/CollectionOverview/collectionState.ts

```diff
@@ -28,7 +33,7 @@ export const useCollectionPubs = (options: UseCollectionPubsOptions) => {
 	} = usePageContext();
 	const { pendingPromise } = usePendingChanges();
 	const [collectionPubs, setCollectionPubs] = useState(() => sortByRank(initialCollectionPubs));
-	const [scopeSummary, setScopeSummary] = useState(collection.scopeSummary);
+	const [scopeSummary, setScopeSummary] = useState<MinimalScopeSummary>(collection.scopeSummary);
```

Comment:
changed this to `MinimalScopeSummary` (see those edits later) bc the `subtractScopeSummaries` and `addScopeSummaries` method do not include `createdAt`, `updatedAt` and `id`.